ext/standard/string.c

```diff
@@ -1030,9 +1030,18 @@ PHP_FUNCTION(implode)
 	ZEND_PARSE_PARAMETERS_START(1, 2)
 		Z_PARAM_ARRAY_HT_OR_STR(arg1_array, arg1_str)
 		Z_PARAM_OPTIONAL
-		Z_PARAM_ARRAY_HT_OR_NULL(pieces)
+		Z_PARAM_ARRAY_HT(pieces)
```

Comment:
Please revert this after the error message change.

Reply:
So we'll always display the "new" message and never "Argument #2 ($array) must be of type array, null given", even if the person explicitly passes `null` as param?

I'm not opposing. I just want to make sure I understand what will be the effect of reverting this. 😅

Reply:
Yes